USER_MANUAL.md

```diff
@@ -336,6 +363,345 @@ _TODO_
+- New Mexico: "ERA5_NM_1x1_hourly_2015_2019.nc"
+
+_TODO_ check if these work
+
+In addition, the following files are included in the GitHub repository:
+
+- "weather_alberta.nc"
+- "weather_marcellus.nc"
+- "weather_permian.nc"
+- "ERA5_2020_2020_Canada_2xRes.nc"
+- "ERA5_2020_2020_US_2xRes.nc"
+
+Each of these files provides hourly weather (wind, temp, precipitation) data spanning the years specified at a spatial resolution of 1 degree latitude and 1 degree longitude. If custom configurations are needed for different regions, spatial resolutions, temporal resolutions, dates, or weather variables (e.g., clouds, snow cover, etc.), they must be downloaded manually from the ERA5 database. The 'ERA5_downloader' python file in the model code folder provides code and guidance for accessing custom weather data.
+
+See [weather_readme](LDAR_Sim/src/weather/weather_readme.md) documentation for further details regarding weather in LDAR-Sim.
+
+**Notes of caution:**
+
+Weather file sizes can become quite large, especially when spatial and temporal resolution increase (maximum resolutions of 1.25 degrees and 1 hour, respectively). Modelers must decide how to navigate these tradeoffs, and understand the implications of the resolutions chosen.
+
+If using different weather files for different programs (e.g., when comparing different regions), weather data must be downloaded manually and saved to the inputs folder before beginning simulations, as the automatic downloader built into LDAR-Sim will only download one file at a time.
+
+### &lt;emissions&gt;
+
+**Description:** This parameter doesn't necessitate user-defined input. Its purpose is to offer a broader categorization for parameters that define the emission characteristics of the virtual world.
+
+**Notes of caution:** The following parameters for defining emissions can be specified at multiple levels of granularity. The values set will always be overwritten by the most detailed level.
+
+_TODO_ maybe link in the flowchart here?
+
+#### &lt;emissions_file&gt;
+
+**Data Type:** String
+
+**Default input:** None
+
+**Description:** This parameter is specifies the name of the csv file that is used to describe the emissions rate characteristics. Refer to the section [Virtual World Defining Files](#10-virtual-world-defining-files) for comprehensive instructions on setting up this file.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** Column headers in the file must correspond to an [emission rate source(ERS)](#ers), [non-repairable emission rate source(NR_ERS)](#nr_ers), or their equivalent when provided in a more granular setting.
+
+#### &lt;ERS&gt;
+
+**Data Type:** String
+
+**Default input:** None
+
+**Description:** The name of a column in the [emissions file](#emissions_file) that is used to parameterize the emissions characteristics of all the general repairable emissions. This parameter will be overwritten by the equivalent value, when provided at a more granular scale.
```

Comment:
This one is better, but same feedback as above - Perhaps we need a better way to describe it here than at a more granular scale.